index.js

```diff
@@ -20,14 +20,6 @@ module.exports = function (a, b) {
 	var aLen = a.length;
 	var bLen = b.length;
 
-	if (aLen === 0) {
-		return bLen;
-	}
-
-	if (bLen === 0) {
```

Comment:
Here aLen is always smaller or equal than bLen due to the swap, i.e. always false (if not or equal to aLen)